R/factset.protobuf.stachextensions/R/V2ColumnOrganizedStachUtilities.R

```diff
@@ -0,0 +1,80 @@
+#' @docType class
+#' @title V2ColumnOrganizedStachUtilities
+#' @description Provide helper functions for column organized stach
+#'
+#' @importFrom R6 R6Class
+#' @export
+V2ColumnOrganizedStachUtilities <- R6::R6Class(
+  "V2ColumnOrganizedStachUtilities",
+  public = list(
+
+    #' @description  Get Primary Table Ids
+    #' @param package  Stach Data which is represented as a Package object
+    #' @return Returns list of primary table ids
+    GetPrimaryTableIds = function(package) {
```

Comment:
As we are using this function only at one place. make this functionality like "package$primary_table_ids" in DecompressAll function.

Reply:
Since the other libraries have this function, I figured to keep it consistent and include this function even though it is one line